File: HistoryService/Entities/ClinicRecord.cs
namespace HistoryService.Entities
{
    public class ClinicRecord
    {
        public string PatientId { get; set; }
        public string DoctorId { get; set; }
        public string Diagnosis { get; set; }
        public Patient Patient { get; set; }
        public Doctor Doctor { get; set; }
    }
}

File: web-app/src/app/core/models/clinicRecord.ts
import { Base } from "./base";
import { Doctor, DoctorRecord } from "./doctor";
import { Patient } from "./patient";

export interface ClinicRecord extends Base{
  doctorId: string;
  patientId: string;
  diagnosis: string;
  patient?: Patient;
  doctor?: DoctorRecord
}


export interface ClinicRecordCreateDto{
  doctorId: string;
  patientId: string;
  diagnosis: string;
}

